src/main/java/uk/gov/hmcts/darts/task/runner/impl/AbstractLockableAutomatedTask.java

```diff
@@ -34,9 +39,23 @@ public abstract class AbstractLockableAutomatedTask implements AutomatedTask {
 
     private final LockingTaskExecutor lockingTaskExecutor;
 
-    protected AbstractLockableAutomatedTask(AutomatedTaskRepository automatedTaskRepository, LockProvider lockProvider) {
+    private final AutomatedTaskConfigurationProperties automatedTaskConfigurationProperties;
+
+    protected AbstractLockableAutomatedTask(AutomatedTaskRepository automatedTaskRepository, LockProvider lockProvider,
+                                            AutomatedTaskConfigurationProperties automatedTaskConfigurationProperties) {
         this.automatedTaskRepository = automatedTaskRepository;
-        lockingTaskExecutor = new DefaultLockingTaskExecutor(lockProvider);
+        this.lockingTaskExecutor = new DefaultLockingTaskExecutor(lockProvider);
+        this.automatedTaskConfigurationProperties = automatedTaskConfigurationProperties;
+    }
+
+    private void setupUserAuthentication() {
+
+        Jwt jwt = Jwt.withTokenValue("automated-task")
+            .header("alg", "RS256")
+            .claim("emails", List.of(automatedTaskConfigurationProperties.getSystemUserEmail()))
```

Comment:
Assuming this approach was suggested by @hemantasharma1129 ?

How do we ensure that we have this secret `user_email_address` configured in the database for the automated task audit stamping? Wondering, do we mean this to be user_account.usr_id=0 currently?

Although perhaps Barry thinks we should have (usr_id=1) for our system user - looking at his standing data script.

Reply:
For the moment, Hemanta said on staging this is to be manually inserted. Once the admin portal is up and running, this will be what gives the functionality to set the email_address properly